src/app/components/organism/timeline-element/timeline-element.component.css

```
@@ -1,324 +1,333 @@
+  contain: layout style;
+}
+
+/* ===========================
+   Item — desktop zig-zag grid
+   =========================== */
 .timeline-item {
-  --item-offset: 33px;
-  --item-offset-right: calc(var(--item-offset) * -1);
-  --item-offset-position: 0%;
+  --slide-origin: -40px;
   position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)